src/Microsoft.AspNetCore.Dispatcher/Patterns/RoutePatternMatcher.cs

```diff
@@ -422,7 +453,20 @@ private bool SavePathSegmentsAsValues(int index, DispatcherValueCollection value
                         }
                         else
                         {
-                            parameterStartIndex = newLastIndex + lastLiteral.RawText.Length;
```

Comment:
Can you do me a favour before you check this in? Do a find-all-references for `RawText`. Are there any other uses?

Reply:
I thought I did that. I'll check again

Reply:
Is this ok? Or I shouldn't use it at all? https://github.com/aspnet/Routing/blob/037e771e9ea0adb523cdee703aaa00df22304166/src/Microsoft.AspNetCore.Dispatcher/Patterns/RoutePatternMatcher.cs#L387